eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md

```diff
@@ -136,15 +136,25 @@ source ./terraform/setup_env.sh
  . terraform\setup_env.ps1
 ```
 
+If you want to run the program in debug mode, you can save the output value. 
+
+![image](resource/output.png)
```

Comment:
it's hard work if we place a png file in each sample, I am wondering whether we can do it just like below
<img width="853" alt="image" src="https://user-images.githubusercontent.com/4465723/158099876-41852903-cfa3-4874-ba64-4452f47f6e8c.png">